src/token-stream.js

```diff
@@ -16,6 +16,21 @@
 
 import { code } from 'esutils';
 
+// following constant is generated by shift-java/scripts/make-unicode-regexen.js, IDENTIFIER_CONTINUE
```

Comment:
Probably refer to this one instead? https://github.com/shapesecurity/shift-parser-js/blob/es2016/scripts/generate-unicode-data.js

Reply:
Also let's just actually save the whole generated file, keep it marked as generated, and import it here.